assets/prototype/application/ui/components/forms/formModal/FormModal.tsx

```diff
@@ -19,21 +29,21 @@ const FormModal = ({ FormComponent, initialValues, onSubmit, onClose, isOpen, ..
 
 	const classes = classNames(Classes.CARD, Classes.ELEVATION_4);
 
-	const overlayStyle = {
+	const overlayStyle: CSSProperties = {
 		boxSizing: 'border-box',
 		maxHeight: '90%',
 		maxWidth: '1200px',
 		minHeight: '50%',
 		minWidth: '320px',
 		width: '80%',
 		overflowY: 'scroll',
-		position: 'absolute',
+		position: 'absolute' as 'absolute',
```

Comment:
¿huh?

No entiendo mi amigo

Reply:
That's casting value `absolute` as type `absolute`

Reply:
oh wow did not know there were types for that...
so just having the correct value is not enough? wow!

Reply:
does that mean that every prop that have it's value as an enum-like string needs to be typed?

Reply:
In certain cases when the type is treated as a `string` instead of the string literal type or a union type you may need to cast that string into literal type.